scipy/interpolate/_ppoly.pyx

```diff
@@ -56,8 +56,8 @@ def evaluate(double_or_complex[:,:,::1] c,
     dx : int
         Order of derivative to evaluate.  The derivative is evaluated
         piecewise and may have discontinuities.
-    extrapolate : int, optional
-        Whether to extrapolate to out-of-bounds points based on first
+    extrapolate : bint, optional
+        Whether to extrapolate to ouf-of-bounds points based on first
```

Comment:
The ouf typo has become endemic, it cannot be eradicated! https://github.com/scipy/scipy/pull/5704